src/testutil/mod.rs

```diff
@@ -0,0 +1,131 @@
+use crate::{leakbox::{LBForthParams, LBForth}, Forth};
+
+pub fn blocking_runtest(contents: &str) {
```

Comment:
probably worth also having a version of this that takes an `&mut Forth`, which this could call --- for the situations where we want to make assertions about the VM state, or test code that also calls other methods on the VM.

Reply:
Added!